lib/systems/include/Controllers/CASEControllerWrapper.h

```diff
@@ -9,10 +9,14 @@ class TorqueControllerCASEWrapper : public virtual Controller
 {
 public:
     TorqueControllerCASEWrapper() = delete;
-    
+    /// @brief makes CASE system apart of Controller hierarchy for use in TC Mux
```

Comment:
Is there a reason this comment is here at the constructor, rather than at the class header? It would make far more sense to describe _what_ a class is when you define the _class_, and makes much less sense to specify _what_ a class is when you're simply _constructing_ an instance.